package.json

```diff
@@ -136,7 +136,7 @@
     "babel-register": "^6.6.0",
     "babel-types": "^6.6.4",
     "babylon": "^6.6.4",
-    "base64-js": "^0.0.8",
+    "base64-js": "^1.0.4",
```

Comment:
Can you split this into a separate PR? PRs that change package.json are not easy to merge: a fb employee needs to update the internal fb npm cache manually.